contracts/Fund.sol

```diff
@@ -22,21 +22,27 @@ contract Fund is Administratable {
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
-  // ========== STATE VARIABLES ==========
+  // ========== STRUCTS & EVENTS ==========
+  
   struct Grant {
     string description;
     uint256 value;
     address recipient;
     bool complete;
   }
 
+  event ManagerChanged(address newManager);
+  event GrantCreated(string grantId, Grant grant);
+  event GrantUpdated(string grantId, Grant grant);
+  event GrantFinalized(string grantId, Grant grant);
+
+  // ========== STATE VARIABLES ==========
+
   address public manager;
   address public admin;
-  Grant[] public grants;
+  string[] public grantIds;
```

Comment:
Added this grantId array so we could still use `getGrantsCount` and also provide grant count in `getSummary`. Let me know if the count is unnecessary and i will remove the array, the `getGrantsCount` method, and the grant count in `getSummary`.

Reply:
Even if the total count is necessary, I think it would be be easier and cheaper to increment/decrement a `uint256`. However, we could also get the count off-chain by just fetching the event logs. I'm not sure of the use case for count so curious to hear @rheeger and @CantelopePeel's thoughts

Reply:
+1... think we can probably get rid of count altogether (as Ben did for claims in Org.sol)

Reply:
Agreed, we do not need an array or even a count to maintain length... we will grab that using events.

Reply:
We were using the count in `-client` to increment & store the index position of a given grant. If we can get a `Grant` by UUID now instead, that removes the need for the counter.

Reply:
Awesome, Ed and I will remove that today and resolve this discussion afterwards